src/LtiMessageLaunch.php

```diff
@@ -153,9 +155,13 @@ public function initialize(array $request)
      */
     public function validate(array $request = null)
     {
-        // @todo remove this in v6.0
-        if ($request === null) {
-            $request = $_POST;
+        // @TODO: Remove this on the next major release.
+        if (!isset($this->request)) {
+            if ($request === null) {
+                $request = $_POST;
+            }
+
+            $this->setRequest($request);
```

Comment:
`!isset()` can mean that the variable is `null` or `undefined`; does the case of `undefined` need to be addressed?